plugins/modules/system/make.py

```diff
@@ -111,6 +116,7 @@ def main():
             chdir=dict(type='path', required=True),
             file=dict(type='path'),
             make=dict(type='path'),
+            jobs=dict(type='int')
```

Comment:
```suggestion
            jobs=dict(type='int'),
```
Makes the next diff when something new is added nicer :)